### PR TITLE
Fut 17 convert app to module

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

File: Future/R/modules.R [hunk 1]
```diff
@@ -0,0 +1,280 @@
+
+
+mainModuleUI <- function(id){
```
Comment (darsoo) [inline Future/R/modules.R:3]: add documentation

File: Future/R/modules.R [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+      )
+   )
+
+   dashboardPage(header, sidebar, body)
+
+}
+
+
+mainModule <- function(input, output, session){
+      shinyjs::addClass(selector = "body", class = "sidebar-collapse")
```
Comment (darsoo) [inline Future/R/modules.R:183]: add comments in code with sections

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 3]
```diff
@@ -1,277 +1,118 @@
...
-            )
-    })
 
-    output$kcalMeal <- renderText({
-        validate(
-            need(rv$out, message = "Wybierz produkt")
-        )
-        paste("Kalorycznosc posilku wynosi", rv$out, "kcal.")
-    })
+# server <- function(input, output, session) {
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:21]: remove commented code

**Review by darsoo — CHANGES_REQUESTED** [review]
Check comments in code

File: Future/R/modules.R (hunk 2 above)
Comment (karollayna) [reply Future/R/modules.R:183]: done

**Review by darsoo — APPROVED** [review]
Update documentation (NAMESPACE) and good to go

**Pull request merged**